dbms/src/Common/FailPoint.cpp

```diff
@@ -103,7 +104,9 @@ std::unordered_map<String, std::shared_ptr<FailPointChannel>> FailPointHelper::f
     M(pause_when_ingesting_to_dt_store)      \
     M(pause_when_altering_dt_store)          \
     M(pause_after_copr_streams_acquired)     \
-    M(pause_before_server_merge_one_delta)
+    M(pause_before_server_merge_one_delta)   \
+    M(pause_query_until_write_finish)        \
+    M(force_pause_query_until_write_finish)
```

Comment:
Seems that this is normal failpoint that does not need channels. You can move it to L88.